kavm/src/kavm/kavm.py

```diff
@@ -166,19 +167,48 @@ def run_avm_json(
             for app in acc['created-apps']:
                 teal_paths.add(app['params']['approval-program'])
                 teal_paths.add(app['params']['clear-state-program'])
-
-        teal_programs: str = ''
+        try:
+            execute_transactions_stage = [
+                stage for stage in avm_json['stages'] if stage['stage-type'] == 'execute-transactions'
+            ][0]
+        except KeyError:
+            print(f'Test file {input_file} does not contain an "execute-transactions" stage')
+            exit(1)
+        for txn in execute_transactions_stage['data']['transactions']:
+            if 'apap' in txn:
+                teal_paths.add(txn['apap'])
+            if 'apsu' in txn:
+                teal_paths.add(txn['apsu'])
+
+        def run_process_on_bison_parser(path: Path) -> str:
+            print(teal_programs_parser)
```

Comment:
This `print` must either be dropped or refactored to use the logger.

Reply:
removed as it was unintentionally left in.